README.md

```diff
@@ -1,194 +1,315 @@
+Julia REPL:
 
-P4est.jl is a registered Julia package. Hence, you can install it by executing
-the following commands in the Julia REPL:
 ```julia
-julia> import Pkg; Pkg.add("P4est")
+julia> import Pkg; Pkg.add(["P4est", "MPI"])
 ```
-P4est.jl depends on the binary distribution of the [p4est](https://github.com/cburstedde/p4est)
-library, which is available in the Julia package P4est\_jll.jl and which is automatically
-installed as a dependency. The binaries provided by P4est\_jll.jl support MPI and are compiled
-against the MPI binaries provided by MicrosoftMPI\_jll.jl on Windows and MPICH\_jll.jl on all
-other platforms. Note that [MPI.jl](https://github.com/JuliaParallel/MPI.jl)
-should be configured to use the same MPI binaries.
-
-By default, P4est.jl provides pre-generated Julia bindings to all exported C
-functions of the underlying p4est library. You can force the build script to
-re-generate the bindings by setting the environment variable
-`JULIA_P4EST_GENERATE_BINDINGS` to a non-empty string.
-
-*Note: Currently, P4est.jl can only be used with pre-generated bindings on
-Julia v1.7. New bindings can only be generated with Julia v1.6.
-See [issue #39](https://github.com/trixi-framework/P4est.jl/issues/39) for further discussions.*
-
-### Using a custom build of p4est
-When `JULIA_P4EST_GENERATE_BINDINGS` is non-empty you can also
-configure P4est.jl to use a custom build of p4est. For this, set the following
-environment variables and build P4est.jl again afterwards:
-1. **Set `JULIA_P4EST_PATH`.**
-
-   You can set the environment variable `JULIA_P4EST_PATH` to the install
-   prefix of your p4est library.
-   P4est.jl will then assume to find the corresponding library as
-   `joinpath(ENV["JULIA_P4EST_PATH"], "lib", "libp4est.{so,dylib,dll}")`
-   and the include files in
-   `joinpath(ENV["JULIA_P4EST_PATH"], "include")`.
-2. **Set `JULIA_P4EST_LIBRARY` and `JULIA_P4EST_INCLUDE`.**
-
-   Alternatively, you can specify the p4est library and the include
-   directory directly. Note that `JULIA_P4EST_LIBRARY` expects the full path to
-   the p4est library, while `JULIA_P4EST_INCLUDE` must be the full path to the
-   directory with the p4est header files.
-
-For example, if your custom p4est build is installed to `/opt/p4est`, you can
-use it from P4est.jl by executing
-```bash
-julia --project -e 'ENV["JULIA_P4EST_GENERATE_BINDINGS"] = "yes";
-                    ENV["JULIA_P4EST_PATH"] = "/opt/p4est";
-                    using Pkg; Pkg.build("P4est"; verbose=true)'
+
+With this command, you install both
+[P4est.jl](https://github.com/trixi-framework/P4est.jl) and
+[MPI.jl](https://github.com/JuliaParallel/MPI.jl).
+Currently, [P4est.jl](https://github.com/trixi-framework/P4est.jl) supports
+only builds of the C library [`p4est`](https://github.com/cburstedde/p4est)
+with MPI support, so you need to initialize MPI appropriately as described
+in the "Usage" section below.
```

Comment:
```suggestion
in the [Usage](#usage) section below.
```

Reply:
Did you check whether this works both in the markdown file rendered on GitHub and with Documenter.jl in the docs (since we copy the `README.md` as `docs/src/index.md`)?

Reply:
Ah. No, I didn't 😬. OK, closing this...

Reply:
Done in d6dad0a